### PR TITLE
Update 4v4 names

### PR DESCRIPTION
**Pull request by BlackYps**
Add keys for leaderboards and convert queue names to snake case

### CODE REVIEW

**Comment by Sheikah45** [comment]
Is this settled?

**Comment by BlackYps** [comment]
I would guess so. There hasn't been any movement in https://github.com/FAForever/db/pull/290 recently. I'm under the impression that it is ready, but the safe bet is probably to wait until that pr is merged

**Review by Sheikah45 — APPROVED** [review]
Damn snake case XD

**Pull request merged**
